src/js/extras/media-clip-selector/index.js

```diff
@@ -19,6 +19,7 @@ function lockBetweenZeroAndOne(num) {
   return Math.max(0, Math.min(1, num));
 }
 
+// @ts-ignore
```

Comment:
I wouldn't expect to have to address all these native properties like this. Wondering if that's gonna get annoying.

Reply:
another option is to turn off `checkJs` in tsc so that it doesn't spew any errors. It's possible there's a way to handling this, but not sure. If I remember correctly, adding `@type {HTMLTemplateElement}` didn't fix it like for the other things.

Reply:
Also, there may be a way of fixing this, but unfortunately, I haven't figured it out yet. It basically comes down to not being able to cast vars like you can in native typescript using the jsdoc syntax.

Reply:
Cool. Not a blocker either way.

Reply:
Apparently, it does support it https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#casts

However, I think that should be a separate PR not to delay this one much longer.